### ISSUE TITLE
Link error on macOS

### ISSUE THREAD

**Issue by Vezzra** [freeorion/freeorion#3500]
Bug Report
==========

Environment
-----------

* **FreeOrion Version**: a52063bf9224295d34a3da8377238227ee0524f6
* **Operating System**: macOS 10.14.6
* **Fetched as** 
  * Compiled from source

Description
-----------

```
ld: can't map file, errno=22 file '/Users/user/SoftwareProjekte/FO/main/build/dep/Frameworks/vorbis.framework' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by geoffthemedio** [comment]
related to https://github.com/freeorion/freeorion/pull/3489 ?

try reverting b9c464b

**Comment by Vezzra** [comment]
Yep, reverting that commit (which, btw, is d79e0d1e9fdce57ca8a83e251c71068e01d13f2e in master) fixes the link error on macOS.

Should the revert be commited?

**Comment by geoffthemedio** [comment]
@sevu can you fix it in ~24 hours?

**Comment by sevu** [comment]
My best hint is 0732058. There's some extra treatment for frameworks … but that applies not to targets, only variables.

Does it work if you exclude the line `Vorbis::VorbisFile` from d79e0d1.

**Comment by Vezzra** [comment]
> Does it work if you exclude the line `Vorbis::VorbisFile` from d79e0d1

No, unfortunately not. Same link error.

As a temporary fix I'm going to revert d79e0d1e9fdce57ca8a83e251c71068e01d13f2e. @sevu, can you try to provide a fixed version of #3489 that works on macOS?

**Comment by Vezzra** [comment]
Fixed by 24a08d6c597ce8afc90ccec485a937982b6c3683 (revert of d79e0d1e9fdce57ca8a83e251c71068e01d13f2e).

**Comment by sevu** [comment]
@Vezzra I made a new PR, could you confirm if it works, after the CI passed.

I'm pretty sure it failed because I had removed the include directory.
On the CI the vorbis installation is probably in a path already included by another dependency, and thus it passed last time.
